ext/lists.go

```diff
@@ -159,6 +174,27 @@ func (lib listsLib) CompileOptions() []cel.EnvOption {
 			),
 		)
 	}
+	if lib.version >= 2 {
+		opts = append(opts,
+			cel.Function("sort",
+				cel.MemberOverload("list_sort",
+					[]*cel.Type{listType}, listType,
```

Comment:
A slight refinement on this approach would be to create a singleton binding for all comparable types: `bool`, `bytes`, `int`, `uint`, `string`, `timestamp`, `duration`. I realize it's a bit annoying but it will really help with ensuring the function is only used with comparable types.

Reply:
I actually created MemberOverloads for `list(int), list(uint), ...` because I couldn't find out how to constrain the element type must be comparable via a `cel.SingletonBinding`. Let me know if there is a better way

Reply:
I see your point, I added helper type traits for list and maps in #1026 to at least better clarify intent. The check you're doing now is quite reasonable.